<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Min</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Max</string>
   </property>
  </widget>
  <widget class="QComboBox" name="uiUnit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>30</y>
     <width>101</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="uiMin">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>62</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-999999999.000000000000000</double>
   </property>
   <property name="maximum">
    <double>999999999.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="uiMax">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>62</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-999999999.000000000000000</double>
   </property>
   <property name="maximum">
    <double>999999999.000000000000000</double>
   </property>
  </widget>
  <widget class="QSpinBox" name="uiSteps">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>60</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-99</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Steps</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
